rfc/text/advanced/ap_50_payload_passthru_mode.md

```diff
@@ -148,12 +149,15 @@ without guessing. The format of the value may depend on the `ppt_scheme` attribu
 
 {align="left"}
 
-| Attribute      | Required? | Value                       |
-|----------------|-----------|-----------------------------|
-| ppt_scheme     | Y         | mqtt                        |
-| ppt_serializer | N         | json, msgpack, cbor, native |
-| ppt_cipher     | N         | -                           |
-| ppt_keyid      | N         | -                           |
+| Attribute      | Required? | Value                        |
+|----------------|-----------|------------------------------|
+| ppt_scheme     | Y         | mqtt                         |
+| ppt_serializer | N*        | json, msgpack, cbor, native* |
+| ppt_cipher     | N         | -                            |
+| ppt_keyid      | N         | -                            |
+
+*: If `ppt_serializer` is not provided then it is assuming as `native`. So no additional serialization 
```

Comment:
the possible values is already defined by the ppt_schemes "wamp.eth" and "mqtt". thus, the don't need to specify what happens if it is "not provided", since it depends on ppt_scheme